packages/react/src/LogoSuite/LogoSuite.figma.tsx

```diff
@@ -0,0 +1,621 @@
+import React from 'react'
+import {LogoSuite} from './LogoSuite'
+import figma from '@figma/code-connect'
+
+/**
+ * -- This file was auto-generated by `figma connect create` --
+ * `props` includes a mapping from Figma properties and variants to
+ * suggested values. You should update this to match the props of your
+ * code component, and update the `example` function to return the
+ * code example you'd like to see in Figma
+ */
+
+figma.connect(LogoSuite, 'https://www.figma.com/design/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=8553-10133&m=dev', {
+  props: {
+    description: figma.textContent('Description'),
+    heading: figma.textContent('Text'),
+    align: figma.enum('align', {
+      start: 'start',
+      center: 'center',
```

Comment:
There's also ['justify'](https://github.com/primer/brand/blob/main/packages/react/src/LogoSuite/LogoSuite.tsx#L26), can we add that too?

```suggestion
    align: figma.enum('align', {
      start: 'start',
      center: 'center',
      justify: 'justify',
```

Reply:
`justify` doesn't exist in the existing Figma component. (It does on the [upcoming version](https://www.figma.com/design/BJ95AjraesmRCWsKA013GS/branch/WFydbNLeRBlwX2E3hTeTVY/Primer-Brand?node-id=8553-10133&m=dev))

Reply:
`justify` doesn't exist in the existing Figma component. (It does on the [upcoming version](https://www.figma.com/design/BJ95AjraesmRCWsKA013GS/branch/WFydbNLeRBlwX2E3hTeTVY/Primer-Brand?node-id=8553-10133&m=dev))

Reply:
It does not exist in Figma